app/models/reservation.rb

```diff
@@ -80,6 +80,10 @@ def self.not_this_reservation(reservation)
     end
   end
 
+  scope :ongoing, -> {
+    where("actual_start_at <= ?", Time.current).where(actual_end_at: nil)
```

Comment:
The `relay_in_progress` would probably work as well. It checks that `actual_start_at IS NOT NULL`. In theory, you won't have an actual start at in the future.